docker/ubuntu16.04-gpu/Dockerfile

```diff
@@ -2,6 +2,8 @@
 FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu16.04
 LABEL maintainer="mdoulaty@gmail.com"
 
+ENV LANG C.UTF-8
```

Comment:
Please don't make this change... the other one is OK.
I prefer all Kaldi scripts to locally set LC_ALL=C to be independent of the system language.

Reply:
.. I mean the code change is OK.

Reply:
OK, I will keep `open(... encoding='utf-8')` and remove `ENV LANG C.UTF-8`.

Reply:
Won't that interfere with "weird" ascii/latin-X/ utterance names?

Reply:
@jtrmal I've decided to just accept that UTF is the only encoding anyone uses.  With our old approach it was a constant battle against Python'd design philosophy.